packages/signers/src/sign-transaction.ts

```diff
@@ -27,7 +27,7 @@ export async function partiallySignTransactionMessageWithSigners<
     TTransactionMessage extends CompilableTransactionMessageWithSigners = CompilableTransactionMessageWithSigners,
 >(transactionMessage: TTransactionMessage, config: { abortSignal?: AbortSignal } = {}): Promise<NewTransaction> {
     const { partialSigners, modifyingSigners } = categorizeTransactionSigners(
-        deduplicateSigners(getSignersFromTransaction(transactionMessage).filter(isTransactionSigner)),
+        deduplicateSigners(getSignersFromTransactionMessage(transactionMessage).filter(isTransactionSigner)),
```

Comment:
Oh never-mind, you're changing it in the cleanup haha.

Reply:
Yea there were a lot of these, so I ended up just fixing it all here since they didn't cause any visible issues. Downside of typescript being so lenient about swapping out identical types that these didn't cause any errors!